playbooks/roles/install-oc/tasks/oc.yml

```diff
@@ -0,0 +1,17 @@
+---
+
+- name: Install required packages
+  yum:
+    name:
+      - "bash-completion"
+    state: "present"
+  register: task_result
+  become: true
+
+- name: Install openshift command-line interface
+  shell: "curl -L 'https://mirror.openshift.com/pub/openshift-v3/clients/{{ openshift_version }}.{{ openshift_patch }}/linux/oc.tar.gz' | tar -xvzf - -C /usr/bin oc &>/dev/null"
```

Comment:
Per ansible docs - /bin/sh default
https://docs.ansible.com/ansible/latest/modules/shell_module.html#shell-module
On most systems /bin/sh is probably a symlink to bash/dash but we shouldn't assume that - change &> shorthand to explicit redirect of stdout and stderr, ie. > /dev/null 2>&1